src/MoveRequestManager.test.js

```diff
@@ -0,0 +1,343 @@
+import {
+  render,
+  screen,
+  fireEvent,
+  cleanup,
+  waitFor,
+} from '@testing-library/react';
+
+import '../test/jest/__mock__';
+
+import MoveRequestManager from './MoveRequestManager';
+import ItemsDialog from './ItemsDialog';
+import ChooseRequestTypeDialog from './ChooseRequestTypeDialog';
+import ErrorModal from './components/ErrorModal';
+import {
+  requestTypesMap,
+} from './constants';
```

Comment:
```suggestion
import { requestTypesMap } from './constants';
```

Reply:
@OleksandrHladchenko1 Fixed.
Could you take a look one more time?